lib/active_model/serializer/version.rb

```diff
@@ -1,5 +1,7 @@
+require 'active_model_serializers/version'
+
 module ActiveModel
   class Serializer
-    VERSION = '0.10.0.rc4'.freeze
+    VERSION = ActiveModelSerializers::VERSION
```

Comment:
Can just be `ActiveModel::Serializer::VERSION =  ActiveModelSerializers::VERSION`. autoload etc takes care of the rest